server/db.js

```diff
@@ -211,4 +212,20 @@ database.insertAnswer = function(taskID, answer, callback) {
     )
 }
 
+database.changeAppointment = function(dateOfAppointment, callback) {
+    database.query(
+        "UPDATE Appointment SET dateOfAppointment = ? WHERE appointmentID = 6;", 
```

Comment:
Is this hardcoded to always delete appointment 6? What happens when appointmentID 6 has already been deleted